src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,102 @@
 import { PeopleFilters } from './PeopleFilters';
```

Comment:
Please, don't forget about imports order. May seem like a tiny thing, but it all sums up at the end :) 
It usually starts with third-party libs, then - more-less depends on a project, but still, there should be a consistent structure, like: 

```
// third-party, starting from react 
import React from 'react'
...
//API
...
//types
...
// components 
```